forge-gui/res/cardsfolder/upcoming/death_kiss.txt

```diff
@@ -0,0 +1,14 @@
+Name:Death Kiss
+ManaCost:5 R
+Types:Creature Beholder
+PT:5/5
+T:Mode$ Attacks | ValidCard$ Creature.OppCtrl | Attacked$ Opponent | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever a creature an opponent controls attacks one of your opponents, double its power until end of turn.
+SVar:TrigPump:DB$ Pump | Defined$ TriggeredAttacker | NumAtt$ +Y | Double$ True
+SVar:Y:TriggeredAttacker$CardPower
+K:Monstrosity:X:X X R
+SVar:X:Count$xPaid
+T:Mode$ BecomeMonstrous | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigGoad | TriggerDescription$ When CARDNAME becomes monstrous, goad up to X target creatures your opponents control.
+SVar:TrigGoad:DB$ Goad | TargetMin$ 0 | TargetMax$ MonstrosityX | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select up to X target creatures your opponents control
+SVar:MonstrosityX:TriggerCount$MonstrosityAmount
+AI:RemoveDeck:Random
+Oracle:Whenever a creature an opponent controls attacks one of your opponents, double its power until end of turn.\n{X}{X}{R}: Monstrosity X. (If this creature isn't monstrous, put X +1/+1 counters on it and it becomes monstrous.)\nWhen Death Kiss becomes monstrous, goad up to X target creatures your opponents control.
```

Comment:
I guess it might benefit from DeckHas:Ability$Counters (might be worth checking if other Monstrosity cards has this specifier, it works well with counter doubling effects etc)